script/color-contrast.ts

```diff
@@ -97,76 +104,80 @@ const testContrast = (minimumContrast: number, colorA: string, colorB: string, b
 }
 /**
  * renderConsoleTable
- * @description takes the test results per theme and prints a nicely formatted table of the results to the console 
- * @param theme 
- * @param results 
+ * @description takes the test results per theme and prints a nicely formatted table of the results to the console
+ * @param theme
+ * @param results
  */
 const renderConsoleTable = (theme: string, results: contrastTestResult[]): void => {
   // config table
   const contrastTable = new Table({
     title: `Contrast checks for: ${theme}`,
-    charLength: { "❌": 2, "✅": 2 },
+    charLength: {'❌': 2, '✅': 2},
     colorMap: {
-      grey: '\x1b[0;30m', // define customized color
+      grey: '\x1b[0;30m' // define customized color
     },
     columns: [
       {
-        name: "contrastPair",
-        alignment: "left",
-        title: "Color pair",
+        name: 'contrastPair',
+        alignment: 'left',
+        title: 'Color pair'
       },
       {
-        name: "pass",
-        alignment: "center",
-        title: "Pass",
+        name: 'pass',
+        alignment: 'center',
+        title: 'Pass'
       },
       {
-        name: "contrastRatio",
-        alignment: "left",
-        title: "Contrast ratio"
+        name: 'contrastRatio',
+        alignment: 'left',
+        title: 'Contrast ratio'
       },
       {
-        name: "minimumContrastRatio",
-        alignment: "left",
-        title: "Min. ratio"
-      }]
+        name: 'minimumContrastRatio',
+        alignment: 'left',
+        title: 'Min. ratio'
+      }
+    ]
   })
   // add rows and color
   for (const [index, row] of results.entries()) {
-    let color = "white"
+    let color = 'white'
     // turn odd index rows grey (index starts at 0)
     if (index % 2 !== 0) {
-      color = "grey"
+      color = 'grey'
     }
     contrastTable.addRow(row, {
-      color: color
+      color
     })
   }
   // print table to console
+  // eslint-disable-next-line no-console
   console.log(contrastTable.render())
 }
+
 /**
  * run tests, output results to console and store them for json
  */
-const results = Object.entries(contrastRequirements)
-  .map(([theme, colorPairs]: [theme: string, colorPairs: ContrastRequirement[]]) => {
-    // turn deeply nested colors object into one level object like 'fg.default': '#000'
-    // @ts-ignore
```

Comment:
@lukasoppermann - I removed the ts-ignore here by switching to an unnamed tuple. You may wanna check you're happy with this.